metrics/test/features/dashboard/presentation/page/dashboard_page_test.dart

```diff
@@ -62,15 +75,33 @@ void main() {
         );
       },
     );
+
+    testWidgets(
+      "Displays the drawer on tap on menu button",
```

Comment:
Should we add test for theme change? After it's changed - colors are changed as well?